python/test/Test_NDItk_MultigroupTable.py

```diff
@@ -133,6 +135,117 @@ def verify_chunk( self, chunk ) :
             self.assertAlmostEqual( 6.0, xs.cross_sections[5] )
             self.assertAlmostEqual( 7.0, xs.cross_sections[6] )
 
+            # verify content - scattering matrix
+            matrix = chunk.scattering_matrix
+            self.assertEqual( "pn_full", matrix.keyword )
+            self.assertEqual( False, matrix.empty )
+            self.assertEqual( 100, matrix.size )
```

Comment:
49 * 2 + 2 moments?

Reply:
Yes: if you have 2 matrices of 7 by 7, the total number of values in the record should by 2 * 7 * 7 + 2 (where these last 2 are the order of the moment)